sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsClient.java

```diff
@@ -175,7 +183,7 @@
 import static com.azure.core.util.FluxUtil.monoError;
 import static com.azure.core.util.tracing.Tracer.AZ_TRACING_NAMESPACE_KEY;
 
-class AnalyzeActionsAsyncClient {
+class AnalyzeActionsClient {
```

Comment:
I'm not sure I fully understand the change to make this client a sync client instead of an async client and to continue having operations which use Reactor.